src/GoogleAuthorize.js

```diff
@@ -64,13 +64,14 @@ class GoogleAuthorize extends Component {
       fetch_basic_profile:    fetchBasicProfile,
       ux_mode:                uxMode,
       redirect_uri:           redirectUri,
+      response_type:          responseType,
       include_granted_scopes: true,
       discoveryDocs,
       prompt,
       scope,
     };
 
-    if (responseType === 'code') {
+    if (params.response_type === 'code') {
       params.access_type = 'offline';
       params.response_type = 'code';
```

Comment:
no need for this duplicate re-assign

Reply:
@MaxwellGBrown any chance to do this, and to also add a test?

Reply:
@eliranamar Thank you; Dropped line 76.

@advance512 Will look into it. Doesn't look as if there's any tests for the click event, and `gapi` isn't mocked out or anything, so I'll see if the effort required matches my desired involvement 😬 .